src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -1201,7 +1201,7 @@ class WalletService(
                         SessionWalletRetrieveResponseDto.OutcomeEnum.NUMBER_0
                     }
                 WalletNotificationRequestDto.OperationResultEnum.AUTHORIZED ->
-                    SessionWalletRetrieveResponseDto.OutcomeEnum.NUMBER_1
+                    SessionWalletRetrieveResponseDto.OutcomeEnum.NUMBER_25
```

Comment:
You should change error code mapping from 1 to 25 also for cards onboarding with DECLINED error code (see `decodeCardsOnboardingNpgErrorCode`, the 3 iso code to outcome mapping).
moreover the default value in case of unhandled conbination should be changed from 1 to 25 too (the null operationResult case into switch)
The main aim here is to remove the generic error whenever possible

Reply:
done here 52c29e53aabcf6781285277d643686a7671faa36

Reply:
![image](https://github.com/user-attachments/assets/66bd3ae4-eef3-4fc6-a189-2f6f47f55b7f)
Are outcome different from 25 in the scope of this task too?

Reply:
I would say yes: the task says to use the mapping table to fix error code mapping so yes, also those codes are to be included

Reply:
ok done here 5bebfbf46dd677f0617cbc961e8a7f693fd0b6d3